Package.swift

```diff
@@ -15,6 +15,10 @@ let package = Package(
             name: "WASI",
             targets: ["WASI"]
         ),
+        .library(
+            name: "WASIBase",
```

Comment:
```suggestion
            name: "WASI",
```

I'd like to make this engine agnostic target as bare `WASI`, and the engine dependent one as `WasmKitWASI`